package.json

```diff
@@ -78,6 +78,8 @@
     "fs-extra": "^9.1.0",
     "git-url-parse": "^11.6.0",
     "globby": "^11.0.4",
+    "https-proxy-agent": "^5.0.0",
+    "is-ip": "^3.1.0",
```

Comment:
Should this be a dev dependency since it's only used in the tests?

Reply:
is_ip at most and its not a big deal, https-proxy-agent is to deal with http proxies

Reply:
`is_ip` was the one I was referring to, simply limiting to a dev dependency. I agree it's "not a big deal" since it is indeed very simple. But why package code that will never run?

Reply:
`setup-cml` uses npm install like we do in the distros. We are not adding the --prod flag.
Probably we should do it.